tests/nested/manual/core20-boot-stable-kernel/task.yaml

```
@@ -0,0 +1,32 @@
+summary: Check snapd built from current branch boots with the stable kernel
```

```suggestion
summary: Check snapd built from current branch boots with the stable fundamental snaps
```

since we are checking the stable base snap, stable kernel snap and stable gadget snap